src/AWS.jl

```diff
@@ -1,235 +1,674 @@
-__precompile__()
-
 module AWS
-using Requests
-using JSON
-using LightXML
-using Compat
-
-const EP_US_EAST_NORTHERN_VIRGINIA     = "ec2.us-east-1.amazonaws.com"
-const EP_US_WEST_OREGON                = "ec2.us-west-2.amazonaws.com"
-const EP_US_WEST_NORTHERN_CALIFORNIA   = "ec2.us-west-1.amazonaws.com"
-const EP_EU_IRELAND                    = "ec2.eu-west-1.amazonaws.com"
-const EP_AP_SINGAPORE                  = "ec2.ap-southeast-1.amazonaws.com"
-const EP_AP_SYDNEY                     = "ec2.ap-southeast-2.amazonaws.com"
-const EP_AP_TOKYO                      = "ec2.ap-northeast-1.amazonaws.com"
-const EP_SA_SAO_PAULO                  = "ec2.sa-east-1.amazonaws.com"
-
-export EP_US_EAST_NORTHERN_VIRGINIA, EP_US_WEST_OREGON, EP_US_WEST_NORTHERN_CALIFORNIA
-export EP_EU_IRELAND, EP_AP_SINGAPORE, EP_AP_SYDNEY, EP_AP_TOKYO, EP_SA_SAO_PAULO
-
-export AWSEnv
-
-const AP_NORTHEAST_1 = "ap-northeast-1" # Asia Pacific (Tokyo) Region
-const AP_SOUTHEAST_1 = "ap-southeast-1" # Asia Pacific (Singapore) Region
-const AP_SOUTHEAST_2 = "ap-southeast-2" # Asia Pacific (Sydney) Region
-const EU_WEST_1 = "eu-west-1" # EU (Ireland) Region
-const SA_EAST_1 = "sa-east-1" # South America (Sao Paulo) Region
-const US_EAST_1 = "us-east-1" # US East (Northern Virginia) Region
-const US_WEST_1 = "us-west-1" # US West (Northern California) Region
-const US_WEST_2 = "us-west-2" # US West (Oregon) Region
-
-function read_aws_config(configFileName::String = joinpath(homedir(), ".aws/config"))
-    global AWS_ID
-    global AWS_SECKEY
-    global AWS_REGION
-
-    isfile(configFileName) || return
-    for line in readlines(configFileName)
-        line = replace(line, " ", "")
-        line = replace(line, "\n", "")
-        segs = split(line, "=")
-        if ismatch(r"^aws_secret_access_key", segs[1])
-            AWS_SECKEY = segs[2]
-        elseif ismatch(r"^aws_access_key_id", segs[1])
-            AWS_ID = segs[2]
-        elseif ismatch(r"^region", segs[1])
-            AWS_REGION = segs[2]
-        end
-    end
+
+using Base64
```

Comment:
Dates is used in this source file but isn't being imported here
```suggestion
using Base64
using Dates
```